README.md

```diff
@@ -109,9 +109,9 @@ cd WhatsApp-Key-Databse-Extractor/
 python helpers/termux_dependencies.py
 ```
 
-4.2 optional, connect adb to your device
+4.2 (optional, already happens in termux_dependencies.py) connect adb to your device and start proot
```

Comment:
starting proot is essential for it to work, I believe I had added that to termux_requirements.py and called it as soon as wa_kdbe.py was called.

Reply:
okay thanks. 
Might be something to include in a termux version of wa_kdbe.py?

Reply:
but the addition is so far okay, isnt it?

Reply:
well tbh I don't think we need a readme title for this, we can implicitly run it inside wa_kdbe.py and let user free of knowing if that thing even exists. It will be easy on him.

Reply:
okay cool! Could you do that? Because I think there would need to be some Termux-detector or something...

Reply:
Yes I can. LoL there's no thing like termux detector, with os.system it return linux but I'll see 2mrw. It's almost 12 in the midnight and I'm calling it a day.

Reply:
oh where you at? Here it's 19:25 (in Europe)

Reply:
India.

Reply:
ok cool